zuul.d/openvswitch-openvswitch-jobs.yaml

```diff
@@ -69,95 +48,32 @@
       ansible_collections_repo: github.com/ansible-collections/openvswitch.openvswitch
 
 - job:
-    name: ansible-test-network-integration-openvswitch-python27
-    parent: ansible-test-network-integration-openvswitch
-    nodeset: openvswitch-2.9.0-python27
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-openvswitch-python27-stable211
-    parent: ansible-test-network-integration-openvswitch
-    nodeset: openvswitch-2.9.0-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_collections: true
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-openvswitch-python27-stable29
-    parent: ansible-test-network-integration-openvswitch
-    nodeset: openvswitch-2.9.0-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_collections: true
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-openvswitch-python35
-    parent: ansible-test-network-integration-openvswitch
-    nodeset: openvswitch-2.9.0-python35
-    vars:
-      ansible_test_python: 3.5
-
-- job:
-    name: ansible-test-network-integration-openvswitch-python36
-    parent: ansible-test-network-integration-openvswitch
-    nodeset: openvswitch-2.9.0-python36
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-openvswitch
```

Comment:
Sorry, I think we should change this to

network-ee-integration-tests-openvswitch-openswitch

where we have

network-ee-integration-tests

followed by collection namespace / name

Just to make things easier on us when we get to collections that have same name but different namespace